sdk/azurestack/ci.mgmt.yml

```diff
@@ -1,9 +1,3 @@
-resources:
-  repositories:
-    - repository: azure-sdk-build-tools
-      type: git
-      name: internal/azure-sdk-build-tools
-
 trigger:
```

Comment:
I assume you didn't switch these to extends because they didn't require any repo references? We definitely should consider converting them all to extends for consistency. @mitchdenny what are your thoughts here?